javascript/chronometer.js

```diff
@@ -1,26 +1,33 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval((currentTime) => this.currentTime++, 1000);
```

Comment:
la función callback dentro de setInterval no lleva ningún parámetro (no es necesario):
`setInterval(() => this.currentTime++, 1000);`